### PR TITLE
Map some stuff in Minecraft, mainly related to Display

### PR DESCRIPTION
**Pull request by exaskye**

### CODE REVIEW

**Comment by shadowfacts** [comment]
LGTM

**Comment by Darkhax** [comment]
LGTM?

File: mappings/net/minecraft/client/Minecraft.mapping [hunk 1]
```diff
@@ -41,18 +43,34 @@ CLASS none/bdo net/minecraft/client/Minecraft
 	METHOD X getSessionService ()Lcom/mojang/authlib/minecraft/MinecraftSessionService;
 	METHOD Z isSnooperEnabled ()Z
 	METHOD a start ()V
+	METHOD a swizzle (Ljava/io/InputStream;)Ljava/nio/ByteBuffer;
+		ARG 0 stream
 	METHOD a scheduleOnMainThread (Ljava/lang/Runnable;)Lcom/google/common/util/concurrent/ListenableFuture;
-	METHOD a (Ljava/lang/String;)V
 	METHOD a setGuiScreen (Lnone/bgk;)V
 		ARG 0 screen
+	METHOD a checkGLError (Ljava/lang/String;)V
```
Comment (asiekierka) [inline mappings/net/minecraft/client/Minecraft.mapping:51]: checkGlError

**Comment by asiekierka** [comment]
Other than that, LGTM

**Pull request merged**